week-3/24-dietary/index.html

```diff
@@ -14,12 +14,37 @@
         <h1>CodeYourFuture Summit</h1>
 
         <p>
-          Let us know if you have any dietary requirements so that we can
-          provide you with the best, yummy food.
+          Let us know if you have any dietary requirements so that we can provide you with the best,
+          yummy food.
         </p>
 
         <form>
           <!-- Your multiple choice goes here -->
+
+          <fieldset>
+            <legend>Please select all of your dietary requirements</legend>
+            <label>
+              <input type="checkbox" />
+              Halal
+            </label>
+            <label>
+              <input type="checkbox" />
+              Kosher
+            </label>
+            <label>
+              <input type="checkbox" />
+              Vegetarian
+            </label>
+            <label>
+              <input type="checkbox" />
+              Vegan
+            </label>
+            <label>
+              <input type="checkbox" />
+              Gluten Free
+            </label>
+          </fieldset>
+
```

Comment:
The checkboxes should usually have a `name` attribute and also a `value` if the form is intended to be submitted to a server.